### PR TITLE
Add comment when updating process with the same version

### PR DESCRIPTION
**Pull request by jedrz**
Just to be sure the process was migrated/synchronized. Currently when there is no difference between processes, on remote enviroment no comment is added, but on local environment a notification is displayed about successful migration.

### CODE REVIEW

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala [hunk 1]
```diff
@@ -78,22 +81,31 @@ class DBProcessRepository(val dbConfig: DbConfig, val modelVersion: ProcessingTy
...
+            .map(_.right.map(_.newVersion))
         }
       }
     }
 
     insertAction
   }
 
-  def updateProcess(updateProcessAction: UpdateProcessAction)(implicit loggedUser: LoggedUser): DB[XError[Option[ProcessVersionEntityData]]] =
+  def updateProcess(updateProcessAction: UpdateProcessAction)(implicit loggedUser: LoggedUser): DB[XError[ProcessUpdated]] = {
```
Comment (lciolecki) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala:94]: I wonder about move adding comment in transaction way at ProcessService. WDYT?

File: ui/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/ProcessPosting.scala [hunk 2]
```diff
@@ -19,9 +19,9 @@ class ProcessPosting {
     toRequest(ProcessConverter.toDisplayable(ProcessCanonizer.canonize(process), TestProcessingTypes.Streaming))
   }
 
-  def toEntityAsProcessToSave(process: EspProcess): RequestEntity = {
+  def toEntityAsProcessToSave(process: EspProcess, comment: String = ""): RequestEntity = {
```
Comment (lciolecki) [inline ui/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/ProcessPosting.scala:22]: `comment` as `Option[String]`?

**Review by lciolecki — APPROVED** [review]
LGTM, but please look at below comments.

File: ui/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/ProcessPosting.scala (hunk 2 above)
Comment (jedrz) [reply ui/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/ProcessPosting.scala:22]: It's required to pass comment in the request body, that's the reason it's mandatory here.

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala (hunk 1 above)
Comment (jedrz) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala:94]: I added an comment about extracting logic.

**Pull request merged**